Terraforming missions epilogue(?): Taking Amy to meet the Wanderers (#9956)
endless-sky/endless-sky: data/human/human missions.txt

mission "Terraforming 13"
	minor
	name "Field Trip"
	description "Amy, who you helped terraform Rand and Tundra, seems to share some of her terraforming philosophy with the Wanderers. Bring her to <destination> to visit them."
	source "Tundra"
	destination "Desi Seledrak"
	passengers 1
	to offer
		has "event: terraforming follow-up"
		has "wanderers sestor done"
	on offer
		conversation

> 			`As you walk through the spaceport, you notice a poster featuring Amy, whom you previously helped terraform this world and Rand. The poster describes that terraforming action, and the long-term goal to restore the warmer climate Tundra had millions of years ago. Amy's work on "equilibrium mapping," the idea that planets naturally "want" to support more complex and varied life, seems similar to the Wanderer philosophy that each planet has its own "will", and that the best way to terraform it is to work with that "will." Amy may be interested in learning about the Wanderers and their techniques. Would you like to find her and tell her about them?`

Reviewer: On line 2508, it is “Amy, who you…”. Here, it is “Amy, whom you…”
Is there a convention that should be followed here?

Author: It says "whom" here because of [this comment](https://github.com/endless-sky/endless-sky/pull/9956#discussion_r1545758912). I don't know which is correct.

> 			`As you walk through the spaceport, you notice a poster featuring Amy. You previously helped her terraform this world and Rand. The poster describes that terraforming action, and the long-term goal to restore the warmer climate Tundra had millions of years ago. Amy's work on "equilibrium mapping," the idea that planets naturally "want" to support more complex and varied life, seems similar to the Wanderer philosophy that each planet has its own "will", and that the best way to terraform it is to work with how that "will." Amy may be interested in learning about the Wanderers and their techniques. Would you like to find her and tell her about them?`

Author: I'm going to apply roadrunner's suggestion on this line so you'll have to redo this after that.
That empty line here doesn't seem appropriate and the second line of text needs to have an indentation at the beginning.

			choice
				`	(Yes.)`
				`	(No.)`

> 					decline

Reviewer: ```suggestion
					defer
```

Author: I've added a separate option to defer.

> 			`	You're able to contact Amy. She has been continuing to monitor the status of Tundra's climate, alongside the researchers from the Deep who she worked with to terraform this world. You tell her you've encountered some alien terraformers who may share her philosophy. She is eager to learn more about them and agrees to meet you in the spaceport bar.`

Author: The first sentence is a bit of a tautology. It doesn't sound too great to me.
"Surprisingly, you're able to contact Amy much quicker than you expected."
Being able to contact her unexpectedly quickly itself implies surprise, since it was unexpected.

But, also, I didn't have any particular expectation on how long it'd take to contact her. Are we trying to make a point that she was exceptionally easy to contact? If so, why?

"a layman summation of your travels since then and briefly inform her" seems kind of verbose, and I don't think it even says what I want it to. Especially in conjunction with the next sentence beginning with: "This catches her attention."
This is the thing I came here to tell her about. It makes it sound like our character is making some long presentation and only when this one thing comes up does she actually seem interested.

> 			`	You're able to contact Amy. She has been continuing to monitor the status of Tundra's climate, alongside the researchers from the Deep who she worked with to terraform this world. You tell her you've encountered some alien terraformers who may share her philosophy. She is eager to learn more about them and agrees to meet you in the spaceport bar.`

Reviewer 2: For point one, I don't have anything against it, and I leave it up to your judgement. 

For the next point, the goal here was to try and link both sentences together better. In the aforementioned text: "You tell her you've encountered some alien terraformers who may share her philosophy. She is eager to learn more about them-" This tells me that, yes, the player tells her about the Wanderer and that she wants to know more. However, the way it's presented feels too barebones. It's a near "action, reaction" sort of text. 

Why I wanted "layman summation" was more to at least give the player some sense that there was some context given, however long the player believes the conversation went, rather than "Hello, there's aliens who've their views on terraforming." I also wanted the text to be more fleshed out, to feel a bit more human, to keep the engagement from the reader.

Perhaps we could go with "quick summary of your adventures?" or something close to that to imply a swift conversation?

> 			`	Once she arrives, you explain about the Wanderers: that the work of their civilisation is to repair worlds left in ruin by their previous inhabitants. And their view that those worlds have their own identities that they must collaborate with them in order for terraforming to be successful.`

Reviewer: ```suggestion
			`	Once she arrives, you explain about the Wanderers: that the work of their civilization is to repair worlds left in ruin by their previous inhabitants. And their view that those worlds have their own identities that they must collaborate with them in order for terraforming to be successful.`
```
American spelling.

Author: Handled by the previous suggestions.

> 			`	Once she arrives, you explain about the Wanderers: that the work of their civilisation is to repair worlds left in ruin by their previous inhabitants. And their view that those worlds have their own identities that they must collaborate with them in order for terraforming to be successful.`

Reviewer 3: ```suggestion
			`	Once she arrives, you explain about the Wanderers: that the work of their civilization is to repair worlds left in ruin by their previous inhabitants. You explain their view that worlds have their own identities that they must collaborate with in order for terraforming to be successful.`
```

> 			`	Once she arrives, you explain about the Wanderers: that the work of their civilisation is to repair worlds left in ruin by their previous inhabitants. And their view that those worlds have their own identities that they must collaborate with them in order for terraforming to be successful.`

Author: "you explain" seems a bit repetitive now.

> 			`	Once she arrives, you explain about the Wanderers: that the work of their civilisation is to repair worlds left in ruin by their previous inhabitants. And their view that those worlds have their own identities that they must collaborate with them in order for terraforming to be successful.`

Author: As with roadrunner's suggestion, I don't like hte sound of the double "you explain".

> 			`	Once she arrives, you explain about the Wanderers: that the work of their civilisation is to repair worlds left in ruin by their previous inhabitants. And their view that those worlds have their own identities that they must collaborate with them in order for terraforming to be successful.`

Reviewer 3: ```suggestion
			`	Once she arrives, you tell her about the Wanderers: that the work of their civilization is to repair worlds left in ruin by their previous inhabitants. You explain their view that those worlds have their own identities that they must collaborate with them in order for terraforming to be successful.`
```

> 			`	Once she arrives, you explain about the Wanderers: that the work of their civilisation is to repair worlds left in ruin by their previous inhabitants. And their view that those worlds have their own identities that they must collaborate with them in order for terraforming to be successful.`

Reviewer 2: Right... I missed that first "you explain." That's on me. But in general, while I abide by the three sentence a paragraph rule religiously, here, it might be best to merge both sentences with the "and."

> 			`	Once she arrives, you explain about the Wanderers: that the work of their civilisation is to repair worlds left in ruin by their previous inhabitants. And their view that those worlds have their own identities that they must collaborate with them in order for terraforming to be successful.`

Author: I've applied this locally with the addition of removing an erroneous "them".

> 			`	She has a slew of questions ranging from how the Wanderers determine a planet's identity to how often asteroids are used in their work. Unfortunately, you are not able to offer much information about the specific details of how the Wanderers go about their terraforming work. Disappointed, but not defeated, Amy asks, "Could you take me to visit these aliens?`

Reviewer: ```suggestion
			`	She has a slew of questions ranging from how the Wanderers determine a planet's identity to how often asteroids are used in their work. Unfortunately, you are not able to offer much information about the specific details of how the Wanderers go about their terraforming work. Disappointed, but not defeated, Amy asks, "Could you take me to visit these aliens?”`
```

Author: Your suggestion contains a "fancy" speech mark; we don't use those.
```suggestion
			`	She has a slew of questions ranging from how the Wanderers determine a planet's identity to how often asteroids are used in their work. Unfortunately, you are not able to offer much information about the specific details of how the Wanderers go about their terraforming work. Disappointed, but not defeated, Amy asks, "Could you take me to visit these aliens?"`
```

> 			`	She has a slew of questions ranging from how the Wanderers' determine a planet's identity to "How often are asteroids used in their work?" Unfortunately, you are not able to offer much information about the specific details of how the Wanderers go about their terraforming work. Disappointed, but not defeated, Amy asks, "Could you take me to visit these aliens?`

Reviewer 3: ```suggestion
			`	She has a slew of questions ranging from how the Wanderers' determine a planet's identity to how often asteroids are used in their work. Unfortunately, you are not able to offer much information about the specific details of how the Wanderers go about their terraforming work. Disappointed, but not defeated, Amy asks, "Could you take me to visit these aliens?`
```

			`	Now that the Kor Mereti drones are peaceful, and the Kor Sestor drones have been eliminated, there is no longer an immediate threat to the Wanderers, and they will be focused much more on their work to repair the former Korath worlds. They would probably be happy to discuss some of that work with someone as enthused by the prospect as Amy.`
			choice
				`	"Alright. I'll prepare a bunk for you."`
				`	"Sorry, that's too far out of my way."`

> 					decline

Reviewer 4: I think this could use a short ending text about the disappointed Amy.

			`	She is delighted, and explains that she will meet you at your ship later today, before leaving to attend to her work in preparation for her absence.`
				accept
	on complete
		conversation

> 			`As you approach <origin>, Amy is astounded by the magnitude of volcanic ash in the air, far outweighing what was released by the volcano you triggered on Tundra. "This place makes Tundra seem temperate," Amy remarks after observing the glaciers that occupy most of the planet's polar regions.`

Reviewer 4: ```suggestion
			`As you approach <planet>, Amy is astounded by the magnitude of volcanic ash in the air, far outweighing what was released by the volcano you triggered on Tundra. "This place makes Tundra seem temperate," Amy remarks after observing the glaciers that occupy most of the planet's polar regions.`
```

			`	Her fascination with the state of this world is temporarily forgotten when she exits your ship and sees the Wanderers for the first time. "I don't know what I was expecting them to be like, but it wasn't birds." However, she quickly returns to the reason for coming here. "Do you know how we can find someone who can answer my questions?"`

> 			`	You approach one of the nearby Wanderers and ask if they could direct you to Tema'a Iriket, the terraforming director. They don't appear to understand the Hai language, but they seem to recognise the name and gesture for you to wait in the spaceport, before heading off in a different direction, presumably to fetch Iriket for you.`

Author: ```suggestion
			`	You approach one of the nearby Wanderers and ask if they could direct you to Tema'a Iriket, the terraforming director. They don't appear to understand the Hai language, but they seem to recognize the name and gesture for you to wait in the spaceport, before heading off in a different direction, presumably to fetch Iriket for you.`
```

> 			`	You approach one of the nearby Wanderers and ask if they could direct you to Tema'a Iriket, the terraforming director. They don't appear to understand the Hai language, but they seem to recognize the name and gesture for you to wait in the spaceport, before heading off in a different direction, presumably to fetch Iriket for you.`

Reviewer 3: My bad.

> 			`	You approach one of the nearby Wanderers and ask if they could direct you to Tema'a Iriket, the terraforming director. They don't appear to understand the Hai language, but they seem to recognize the name and gesture for you to wait in the spaceport, before heading off in a different direction, presumably to fetch Iriket for you.`

Reviewer 2: ```suggestion
			`	Approaching one of the nearby Wanderers, you ask if they could direct you to Tema'a Iriket, the terraforming director, though they don't appear to understand the Hai language from your translator. Still, they seem to recognize the name and gesture for you to wait in the spaceport, flying off into the distance presumably to fetch Iriket for you.`
```

Author: Why do you suggest rearranging the sentences in this way?
I don't think the part about the Wanderer not understanding the Hai translator fits better with the preceding clause.
First, I think it more closely relates to the following clause, since that is about how we overcome that issue, following on directly from it.
Second, grouping it with the preceding clause in one sentence gives me a sense of concurrency, we ask them though they don't understand what we say, but we aren't going to know that they don't understand until after we've asked, or at least, until after outr character is done speaking and the translator begins. It gives the sense that we ask despite them not understanding, but we haven't gone into it knowing that they won't understand. I think it works better in the next sentence where it becomes more like "they recognise the name despite not understanding the Hai language."

Additionally, I think the "before" is important in the last part because, as you have it, it seems like they are flying off as they gesture towards the spaceport, which I don't think works very well. I'm envisioning them gesture and then fly off; two separate actions.

I have applied some of the other adjustments here, though.

mission "Terraforming 14"
	name "Field Trip"
	description "Take Tema'a Iriket and Amy to <destination> to show her a world where the Wanderers have finished their work."
	source "Desi Seledrak"
	destination "Var' Kayi"
	passengers 2
	blocked "You need <capacity> in order to take on this mission."
	to offer
		has "Terraforming 13: done"
	on offer
		conversation

> 			`It is not long before Iriket arrives. You introduce Amy to her, and explain why you have brought her here. "I would be [glad, honoured] to answer any questions about our work," she agrees.`

Author: ```suggestion
			`It is not long before Iriket arrives. You introduce Amy to her, and explain why you have brought her here. "I would be [glad, honored] to answer any questions about our work," she agrees.`
```

> 			`	What follows is an extensive discussion about how the Wanderers go about planning the terraforming process for a world. From the earliest considerations as to what the goal for each planet should be to the role of their artificial minds in helping to monitor and guide the process in the long term. Your translator has some difficulty with the more technical aspects of the discussion, which leads Iriket to rephrase some things many times. Throughout it all, Amy is taking copious notes. Iriket also explains some of what they have been doing to improve the climate on this world.`

Author: Does this have to be one sentence? How about:
```suggestion
			`	What follows is an extensive discussion about how the Wanderers go about planning the terraforming process for a world. Topic range from the earliest considerations as to what the goal for each planet should be to the role of their artificial minds in helping to monitor and guide the process in the long term. Your translator has some difficulty with the more technical aspects of the discussion, which leads Iriket to rephrase some things many times. Throughout it all, Amy is taking copious notes. Iriket also explains some of what they have been doing to improve the climate on this world.`
```

> 			`	What follows is an extensive discussion about how the Wanderers go about planning the terraforming process for a world. From the earliest considerations as to what the goal for each planet should be to the role of their artificial minds in helping to monitor and guide the process in the long term. Your translator has some difficulty with the more technical aspects of the discussion, which leads Iriket to rephrase some things many times. Throughout it all, Amy is taking copious notes. Iriket also explains some of what they have been doing to improve the climate on this world.`

Reviewer 3: Sounds good, but `Topic` should be `Topics`.
```suggestion
			`	What follows is an extensive discussion about how the Wanderers go about planning the terraforming process for a world. Topics range from the earliest considerations as to what the goal for each planet should be to the role of their artificial minds in helping to monitor and guide the process in the long term. Your translator has some difficulty with the more technical aspects of the discussion, which leads Iriket to rephrase some things many times. Throughout it all, Amy is taking copious notes. Iriket also explains some of what they have been doing to improve the climate on this world.`
```

> 			`	"The gases [released, produced] by the factories will help the atmosphere retain more heat, which will in turn begin to melt the glaciers and return water to the [ecosystem, environment]. Over time, this will [mix, interact] with the abundant ash, helping to [restore, rejuvenate] the soils all over this world. In time, it will become [fruitful, bountiful] again."`

Reviewer 2: ```suggestion
			`	"The gases [released, produced] by the factories will help the atmosphere retain more heat, which will, in turn, begin to melt the glaciers and return water to the [ecosystem, environment]. Over time, this will [mix, interact] with the abundant ash, helping to [restore, rejuvenate] the soils worldwide. In time, it will become [fruitful, bountiful] again."`
```

Author: The final sentence here says "it," referring to "this world." I don't think that works if we say "worldwide."

Reviewer 2: Hmm. I was under the impression that "it" referred to the soil.

Reviewer: It could be seen both ways, so it might be nice to change it slightly.

Author: Given that both work and have essentially the same meaning, I don't know what change would be necessary.

			`	"Won't that take hundreds or thousands of years?" Amy asks.`
			`	"Indeed," says Iriket. "But, to attempt to [accelerate, advance] the process by more [direct?] means would [harm, deteriorate] the product. While some worlds may [benefit from, require] a more [extreme, radical] change to begin their restoration, we do not believe it is [necessary, appropriate] here. It is [important, our goal] that a world at the conclusion of our work not only be [pristine, healthy] in that moment, but that it be [sustainable?] without additional [intervention, support]."`

> 			`	Eventually, Amy asks "Are there any worlds where you have completed your work that I could see?" Iriket agrees and asks you to take them to <destination>.`

Author: She's not really agreeing to letting her see that world, I can just take her there. She's agreeing to come with, a sort of tour guide, and suggesting this planet in particular.
```suggestion
			`	Eventually, Amy asks "Are there any worlds where you have completed your work that I could see?" Iriket suggests <destination>, and asks you to take them there.`
```

			choice
				`	"Sure."`
					accept

> 				`	"Sorry, I need to go elsewhere."`

Reviewer 4: And you just leave her in the middle of alien space, taking the only translating device? That sounds like fun.

Author: I removed the choice.

					decline

mission "Terraforming 15"
	landing
	name "Field Trip"
	description "Take Tema'a Iriket and Amy to <destination> to retrieve records of Wanderer terraforming for Amy to study."
	source "Var' Kayi"
	destination "Vara K'chrai"
	passengers 2
	to offer
		has "Terraforming 14: done"
	on offer
		conversation
			`You fly across large swathes of the planet's surface, giving Amy a view of all the greenery and the oceans here. Meanwhile, Tema'a Iriket describes the condition of this world when the Wanderers first began working on it. "The atmosphere was [close to, nearing] a [runaway, uncontrolled] greenhouse effect. Many species had recently gone extinct, or were very close to it. The oceans were also quite toxic. Now, the atmosphere is [stable, sustainable], and many of the species that were [on the brink?] of extinction now flourish."`
			`	Amy also explains about her recent experimental terraforming. "We believe the planet used to be temperate, or even tropical millions of years ago. We used an asteroid impact to trigger a supervolcano that was close to eruption. The greenhouse gases from the eruption have already begun to warm the climate."`

> 			`	Iriket thinks for a moment before responding. "Using the planet's [features, assets] to restore more habitable conditions is an [admirable?] approach." The approval buoys Amy, but she is somewhat deflated by what comes next. "However, an asteroid would not be our [preferred, first] step." `

Reviewer 3: Perhaps elaborate on the rest of this conversation?
```suggestion
			`	Iriket thinks for a moment before responding. "Using the planet's [features, assets] to restore more habitable conditions is an [admirable?] approach." The approval buoys Amy, but she is somewhat deflated by what comes next. "However, an asteroid would not be our [preferred, first] step."`
```

			`	As you conclude your sightseeing, Tema'a Iriket says your next stop should be <planet>, where she will provide Amy with more detailed records of their terraforming activities than she can provide orally.`

> 			`	Once Iriket has returned to her bunk, Amy says to you "I wonder what my professor would say about 'anthropamorphising planets' if he saw this," she gestures towards the images of <origin> on the monitors, "and you told him they believe planets have their own identities, too."`

Reviewer 3: ```suggestion
			`	Once Iriket has returned to her bunk, Amy says to you, "I wonder what my professor would say about 'anthropomorphizing planets' if he saw this," she gestures towards the images of <origin> on the monitors, "and you told him they believe planets have their own identities, too."`
```

				accept
	on complete
		dialog "When you arrive on <planet>, Tema'a Iriket tells you to meet her in the spaceport in a few hours before heading off to prepare the data for Amy."

mission "Terraforming 16"
	name "Field Trip"
	description "Return Amy to <destination> following her trip to meet the Wanderers and learn about how they go about terraforming."
	source "Vara K'chrai"

> 	destination "Tundra"

Reviewer: Do you need quotation marks here?

	passengers 1
	to offer
		has "Terraforming 15: done"
	on offer
		conversation

> 			`Tema'a Iriket meets you in the spaceport and hands Amy a tablet, explaining "I have stored [records, histories] of over a hundred worlds from our archive here. I hope the translation is [adequate, acceptable]. There has not been much opportunity to test our translation systems for the human language."`

Reviewer 2: ```suggestion
			`Tema'a Iriket meets you in the spaceport and hands Amy a tablet, explaining; "I have stored [records, histories] of over a hundred worlds from our archive here. I hope the translation is [adequate, acceptable]. There has not been much opportunity to test our translation systems for the human language."`
```

Author: I don't think we ever use a semi-colon to introduce speech.
```suggestion
			`Tema'a Iriket meets you in the spaceport and hands Amy a tablet, explaining, "I have stored [records, histories] of over a hundred worlds from our archive here. I hope the translation is [adequate, acceptable]. There has not been much opportunity to test our translation systems for the human language."`
```

			`	Amy has already begun browsing the data; Iriket's concerns about the quality of the translation seem to go unheard. "This is incredible. Thank you so much."`

> 			`	"Explaining our work to an outsider has been [gratifying, novel]. I must thank you for the opportunity, captain <last>." Iriket bids you goodbye.`

Reviewer 2: ```suggestion
			`	"Explaining our work to an outsider has been [gratifying, novel]. I must thank you for the opportunity, captain <last>," Iriket says, biding you goodbye as you guide Amy towards the ramp.`
```

> 			`	"Explaining our work to an outsider has been [gratifying, novel]. I must thank you for the opportunity, Captain <last>." Iriket bids you goodbye.`

Reviewer: ```suggestion
			`	"Explaining our work to an outsider has been [gratifying, novel]. I must thank you for the opportunity, Captain <last>," Iriket bids you goodbye.`
```
Should it be done this way, to avoid stubs?

Author: I'm not sure what you mean; "Iriket bids you goodbye" is a complete sentence.

> 			`	Amy spends the rest of your time here in her bunk, studying the Wanderers' data. It seems it's time to return her to <planet>.`

Reviewer 2: Sounds very similar to the first paragraph for the "on complete" conversation. Requires a rewrite.

Author: I changed this line.

> 			`	Amy spends the rest of your time here in her bunk, studying the Wanderers' data. It seems it's time to  return her to <planet>.`

Author: ```suggestion
			`	Amy spends the rest of your time here in her bunk, studying the Wanderers' data. It seems it's time to return her to <planet>.`
```

				accept
	on complete
		conversation

> 			`	Most of Amy's time on the return journey was spent with the Wanderer data, even still, she claims to have barely scratched the surface of what the Wanderers have done.`

Reviewer 3: ```suggestion
			`	Most of Amy's time on the return journey was spent with the Wanderer data. Even still, she claims to have barely scratched the surface of what the Wanderers have done.`
```

			choice
				`	"I'll leave you to your work here."`
					goto done
				`	"What did you think of the Wanderers?"`

> 			`	"Terraforming for the sake of improving ecosystems, providing a place for future inhabitants, is very different to most of what I saw in the Paradise worlds." She explains. "Many of the people there are only interested in terraforming for the economic opportunities.`

Author: I've tried to incorporate some of these ideas, but also, Amy isn't a conduit for an objective, well-rounded summary of the Paradise world terraforming practices that considers all viewpoints, she does think they are greedy. Perhaps a separate opportunity to see it from the other side is in order.

Also, the "suggestion" in your comment doesn't appear to make any changes; is this correct?

> 			`	"I've heard about a project in the Free Worlds working on affordable terraforming as well. Perhaps we should look into working with them. The Republic might not be too happy about that, though."`

Reviewer: You could add something to see if you’ve done Reconciliation or Checkmate here. If first last has done Reconciliation, the Republic would probably be fine with it if their ally gets better ways to support themselves.

			choice
				`	"I'm glad you had a good time."`
					goto done
				# TODO: when alternatives to the Free Worlds are available, this will need to be conditional.
				`	"I actually worked with the founder of the Conservatory. I'm sure I could introduce you."`

> 			`	She thanks you for the offer but declines. "You've done more than enough for now. Besides, I'm going to be busy studying for the forseeable future."`

Author: ```suggestion
			`	She thanks you for the offer but declines. "You've done more than enough for now. Besides, I'm going to be busy studying for the foreseeable future."`
```

mission "Terraforming 13"
	minor
	name "Field Trip"

> 	description "Amy, who you helped terraform Rand and Tundra, seems to share some of her terraforming philosophy with the Wanderers. Bring her to <destination> to visit them."

Reviewer: ```suggestion
	description "Amy, whom you helped terraform Rand and Tundra, seems to share some of her terraforming philosophy with the Wanderers. Bring her to <destination> to visit them."
```

	source "Tundra"
	destination "Desi Seledrak"
	passengers 1
	to offer
		has "event: terraforming follow-up"
		has "wanderers sestor done"
	on offer
		conversation
			branch repeat
				has "terraforming field trip: deferred later"

> 			`As you walk through the spaceport, you notice a poster featuring Amy, whom you previously helped terraform this world and Rand. The poster describes that terraforming action, and the long-term goal to restore the warmer climate Tundra had millions of years ago. Amy's work on "equilibrium mapping," the idea that planets naturally "want" to support more complex and varied life, seems similar to the Wanderer philosophy that each planet has its own "will", and that the best way to terraform it is to work with that "will." Amy may be interested in learning about the Wanderers and their techniques. Would you like to find her and tell her about them?`

Reviewer: ```suggestion
			`As you walk through the spaceport, you notice a poster featuring Amy, whom you previously helped terraform this world and Rand. The poster describes that terraforming action, and the long-term goal to restore the warmer climate Tundra had millions of years ago. Amy's work on "equilibrium mapping," the idea that planets naturally "want" to support more complex and varied life, seems similar to the Wanderer philosophy that each planet has its own "will," and that the best way to terraform it is to work with that "will." Amy may be interested in learning about the Wanderers and their techniques. Would you like to find her and tell her about them?`
```

			choice
				`	(Yes.)`
				`	(Not right now.)`
					defer
				`	(No.)`
					decline
			`	The poster includes small writing with contact instructions for inquiries about the terraforming process and Tundra's climate. Once you get through to a person on the other side, there is initially reluctance to put you through to anyone involved with the terraforming directly, but after verifying that you worked with them to get the process started, you are connected to Amy.`

> 			`	You briefly summarize your adventures, including about the Wanderers: "Aliens with an approach to terraforming that matches your theories from when you were at the Academy of Planetary Sciences." She seems interested in learning more and agrees to meet you in the spaceport bar.`

Reviewer: Including what about the Wanderers? Maybe remove the word "about"?

Author: The text that follows is what about the Wanderers we tell her.

mission "Terraforming 14"
	name "Field Trip"
	description "Take Tema'a Iriket and Amy to <destination> to show her a world where the Wanderers have finished their work."
	source "Desi Seledrak"
	destination "Var' Kayi"
	passengers 2
	blocked "You need <capacity> in order to take on this mission."
	to offer
		has "Terraforming 13: done"
	on offer
		conversation
			`It is not long before Iriket arrives. You introduce Amy to her, and explain why you have brought her here. "I would be [glad, honored] to answer any questions about our work," she agrees.`
			`	What follows is an extensive discussion about how the Wanderers go about planning the terraforming process for a world. Topics range from the earliest considerations as to what the goal for each planet should be to the role of their artificial minds in helping to monitor and guide the process in the long term. Your translator has some difficulty with the more technical aspects of the discussion, which leads Iriket to rephrase some things many times. Throughout it all, Amy is taking copious notes. Iriket also explains some of what they have been doing to improve the climate on this world.`

> 			`	"The gases [released, produced] by the factories will help the atmosphere retain more heat, which will in turn begin to melt the glaciers and return water to the [ecosystem, environment]. Over time, this will [mix, interact] with the abundant ash, helping to [restore, rejuvenate] the soils all over this world. In time, it will become [fruitful, bountiful] again."`

Reviewer: ```suggestion
			`	"The gasses [released, produced] by the factories will help the atmosphere retain more heat, which will in turn begin to melt the glaciers and return water to the [ecosystem, environment]. Over time, this will [mix, interact] with the abundant ash, helping to [restore, rejuvenate] the soils all over this world. In time, it will become [fruitful, bountiful] again."`
```
American spelling

Author: Source? It seems like "gasses" is usually used for the verb form, and the plural noun is usually "gases". I can't find evidence that the American standard for the noun form is "gasses".

Reviewer: Further investigation: ~~why did I take my source from only two websites~~ this can be ignored.

			`	"Won't that take hundreds or thousands of years?" Amy asks.`
			`	"Indeed," says Iriket. "But, to attempt to [accelerate, advance] the process by more [direct?] means would [harm, deteriorate] the product. While some worlds may [benefit from, require] a more [extreme, radical] change to begin their restoration, we do not believe it is [necessary, appropriate] here. It is [important, our goal] that a world at the conclusion of our work not only be [pristine, healthy] in that moment, but that it be [sustainable?] without additional [intervention, support]."`

> 			`	Eventually, Amy asks "Are there any worlds where you have completed your work that I could see?" Iriket suggests <destination>, and asks you to take them there.`

Reviewer: ```suggestion
			`	Eventually, Amy asks "Are there any worlds where you have completed your work that I could see?" Iriket suggests <destination>, and asks you to take them there.`
				accept	
```
Though I think a comma might be warranted:
```suggestion
			`	Eventually, Amy asks, "Are there any worlds where you have completed your work that I could see?" Iriket suggests <destination>, and asks you to take them there.`
				accept
```

mission "Terraforming 15"
	landing
	name "Field Trip"
	description "Take Tema'a Iriket and Amy to <destination> to retrieve records of Wanderer terraforming for Amy to study."
	source "Var' Kayi"
	destination "Vara K'chrai"
	passengers 2
	to offer
		has "Terraforming 14: done"
	on offer
		conversation

> 			`You fly across large swathes of the planet's surface, giving Amy a view of all the greenery and the oceans here. Meanwhile, Tema'a Iriket describes the condition of this world when the Wanderers first began working on it. "The atmosphere was [close to, nearing] a [runaway, uncontrolled] greenhouse effect. Many species had recently gone extinct, or were very close to it. The oceans were also quite toxic. Now, the atmosphere is [stable, sustainable], and many of the species that were [on the brink?] of extinction now flourish."`

Reviewer: I'm finding trouble verifying this, but would the American spelling be swaths (without the e)?

Author: ```suggestion
			`You fly across large swaths of the planet's surface, giving Amy a view of all the greenery and the oceans here. Meanwhile, Tema'a Iriket describes the condition of this world when the Wanderers first began working on it. "The atmosphere was [close to, nearing] a [runaway, uncontrolled] greenhouse effect. Many species had recently gone extinct, or were very close to it. The oceans were also quite toxic. Now, the atmosphere is [stable, sustainable], and many of the species that were [on the brink?] of extinction now flourish."`
```

mission "Terraforming 16"
	name "Field Trip"
	description "Return Amy to <destination> following her trip to meet the Wanderers and learn about how they go about terraforming."
	source "Vara K'chrai"
	destination "Tundra"
	passengers 1
	to offer
		has "Terraforming 15: done"
	on offer
		conversation
			`Tema'a Iriket meets you in the spaceport and hands Amy a tablet, explaining, "I have stored [records, histories] of over a hundred worlds from our archive here. I hope the translation is [adequate, acceptable]. There has not been much opportunity to test our translation systems for the human language."`
			`	Amy has already begun browsing the data; Iriket's concerns about the quality of the translation seem to go unheard. "This is incredible. Thank you so much."`
			`	"Explaining our work to an outsider has been [gratifying, novel]. I must thank you for the opportunity, Captain <last>." Iriket bids you goodbye.`
			`	Amy returns to her bunk, beginning to organize the notes she has made on this trip. It seems it's time to return her to <planet>.`
				accept
	on complete
		conversation

> 			`	Most of Amy's time on the return journey was spent with the Wanderer data. Even still, she claims to have barely scratched the surface of what the Wanderers have done.`

Reviewer: ```suggestion
			`Most of Amy's time on the return journey was spent with the Wanderer data. Even still, she claims to have barely scratched the surface of what the Wanderers have done.`
```

			choice
				`	(Yes.)`
				`	(Not right now.)`
					defer
				`	(No.)`
					decline
			`	The poster includes small writing with contact instructions for inquiries about the terraforming process and Tundra's climate. Once you get through to a person on the other side, there is initially reluctance to put you through to anyone involved with the terraforming directly, but after verifying that you worked with them to get the process started, you are connected to Amy.`
			`	You briefly summarize your adventures, including about the Wanderers: "Aliens with an approach to terraforming that matches your theories from when you were at the Academy of Planetary Sciences." She seems interested in learning more and agrees to meet you in the spaceport bar.`

> 			`	Once she arrives, you tell her about the Wanderers: that the work of their civilization is to repair worlds left in ruin by their previous inhabitants. You explain their view that those worlds have their own identities that they must collaborate with in order for terraforming to be successful.`

Author: That would certainly explain why the comment is still open and unresponded to.

> 			`	Now that the Kor Mereti drones are peaceful, and the Kor Sestor drones have been eliminated, there is no longer an immediate threat to the Wanderers, and they will be focused much more on their work to repair the former Korath worlds. They would probably be happy to discuss some of that work with someone as enthused by the prospect as Amy.`

Author: Looks like a line between these last two got deleted for some reason.
I'll put it back.

> 			`	Now that the Kor Mereti drones are peaceful, and the Kor Sestor drones have been eliminated, there is no longer an immediate threat to the Wanderers, and they will be focused much more on their work to repair the former Korath worlds. They would probably be happy to discuss some of that work with someone as enthused by the prospect as Amy.`

Author: Does https://github.com/endless-sky/endless-sky/pull/9956/commits/bc02a001d8986063b3899bc81cf8c8843593b49b make this make sense?

			choice
				`	"Alright. I'll prepare a bunk for you."`
					goto end
				`	"Sorry, that's too far out of my way."`
			action
				set "terraforming field trip: deferred later"

> 			`	Amy's disappointment returns, but before leaving, she gives you her contact information, saying, "Well, I'd still love to come along the next time you are travelling that way."`

Author: Also a victim of the erroneous deletion.
Does https://github.com/endless-sky/endless-sky/pull/9956/commits/bc02a001d8986063b3899bc81cf8c8843593b49b make this make sense?

				accept

mission "Terraforming Field Trip 3"
	landing
	name "Field Trip"
	description "Take Tema'a Iriket and Amy to <destination> to retrieve records of Wanderer terraforming for Amy to study."
	source "Var' Kayi"
	destination "Vara K'chrai"
	passengers 2
	to offer
		has "Terraforming Field Trip 2: done"
	on offer
		conversation
			`You fly across large swaths of the planet's surface, giving Amy a view of all the greenery and the oceans here. Meanwhile, Tema'a Iriket describes the condition of this world when the Wanderers first began working on it. "The atmosphere was [close to, nearing] a [runaway, uncontrolled] greenhouse effect. Many species had recently gone extinct, or were very close to it. The oceans were also quite toxic. Now, the atmosphere is [stable, sustainable], and many of the species that were [on the brink?] of extinction now flourish."`
			`	Amy also explains about her recent experimental terraforming. "We believe the planet used to be temperate, or even tropical millions of years ago. We used an asteroid impact to trigger a supervolcano that was close to eruption. The greenhouse gases from the eruption have already begun to warm the climate."`

> 			`	Iriket thinks for a moment before responding. "Using the planet's [features, assets] to restore more habitable conditions is an [admirable?] approach." The approval buoys Amy, but she is somewhat deflated by what comes next. "However, an asteroid would not be our [preferred, first] step."`

Author: Why?

> 			`	Iriket thinks for a moment before responding. "Using the planet's [features, assets] to restore more habitable conditions is an [admirable?] approach." The approval buoys Amy, but she is somewhat deflated by what comes next. "However, an asteroid would not be our [preferred, first] step."`

Reviewer 3: 'preferred method' sounds kind of weird. Either way I'm not super crazy about this change, if you want to ignore it that's fine.

> 			`	Iriket thinks for a moment before responding. "Using the planet's [features, assets] to restore more habitable conditions is an [admirable?] approach." The approval buoys Amy, but she is somewhat deflated by what comes next. "However, an asteroid would not be our [preferred, first] step."`

Author: What do you think of "[preferred, first] approach"?

> 			`	Iriket thinks for a moment before responding. "Using the planet's [features, assets] to restore more habitable conditions is an [admirable?] approach." The approval buoys Amy, but she is somewhat deflated by what comes next. "However, an asteroid would not be our [preferred, first] step."`

Reviewer 3: That sounds good.

> 			`	Iriket thinks for a moment before responding. "Using the planet's [features, assets] to restore more habitable conditions is an [admirable?] approach." The approval buoys Amy, but she is somewhat deflated by what comes next. "However, an asteroid would not be our [preferred, first] step."`

Author: Wait, I read this wrong. I thought it is already "method" and you were changing it to "step".
I like "preferred method" and "first step" over the other combinations, but I think "approach" works both ways.
```suggestion
			`	Iriket thinks for a moment before responding. "Using the planet's [features, assets] to restore more habitable conditions is an [admirable?] approach." The approval buoys Amy, but she is somewhat deflated by what comes next. "However, an asteroid would not be our [preferred, first] approach."`
```

			`	Amy has already begun browsing the data; Iriket's concerns about the quality of the translation seem to go unheard. "This is incredible. Thank you so much."`
			`	"Explaining our work to an outsider has been [gratifying, novel]. I must thank you for the opportunity, Captain <last>." Iriket bids you goodbye.`
			`	Amy returns to her bunk, beginning to organize the notes she has made on this trip. It seems it's time to return her to <planet>.`
				accept
	on complete
		log `Took Amy on a "field trip" to Wanderer space to learn about their approach to terraforming. Tema'a Iriket provided great insight and showed Amy both their ongoing and complete work, as well as providing an extensive library of Wanderer records that should keep Amy busy for a very long time.`
		conversation
			`Most of Amy's time on the return journey was spent with the Wanderer data. Even still, she claims to have barely scratched the surface of what the Wanderers have done.`
			choice
				`	"I'll leave you to your work here."`
					goto done
				`	"What did you think of the Wanderers?"`

> 			`	"Terraforming for the sake of improving ecosystems, providing a place for future inhabitants, whoever they may be, is very different to most of what I saw in the Paradise worlds," she explains. "Many of the people there are only interested in terraforming for the economic opportunities for themselves, and they have little regard for anything that doesn't directly increase the value for themselves or their clients. Even the garden or 'natural' areas are artificial; forcing the whims and desires of the rich onto nature.`

Reviewer 3: ```suggestion
			`	"Terraforming for the sake of improving ecosystems, providing a place for future inhabitants, whoever they may be, is very different to most of what I saw in the Paradise Worlds," she explains. "Many of the people there are only interested in terraforming for the economic opportunities for themselves, and they have little regard for anything that doesn't directly increase the planet's value for themselves or their clients. Even the garden or 'natural' areas are artificial; forcing the whims and desires of the rich onto nature.`
```

mission "Terraforming Field Trip 1"
	minor
	name "Field Trip"
	description "Amy, whom you helped terraform Rand and Tundra, seems to share some of her terraforming philosophy with the Wanderers. Bring her to <destination> to visit them."
	source "Tundra"
	destination "Desi Seledrak"
	passengers 1
	to offer
		has "event: terraforming follow-up"
		has "wanderers sestor done"
	on offer
		conversation
			branch repeat

> 				has "terraforming field trip: deferred later"

Reviewer: ```suggestion
				has "terraforming field trip: deferred for later"
```
I'm not sure how much we care about conditions being correct, but I don't think it'd hurt to change.

Author: This isn't what the condition means.
The mission can be deferred at two points. First when the narrator asks you if you want to find her, and second when Amy asks to visit the Wanderers.
This condition is set at the second, later point.

			`As you walk through the spaceport, you notice a poster featuring Amy, whom you previously helped terraform this world and Rand. The poster describes that terraforming action, and the long-term goal to restore the warmer climate Tundra had millions of years ago. Amy's work on "equilibrium mapping," the idea that planets naturally "want" to support more complex and varied life, seems similar to the Wanderer philosophy that each planet has its own "will," and that the best way to terraform it is to work with that "will." Amy may be interested in learning about the Wanderers and their techniques. Would you like to find her and tell her about them?`
			choice
				`	(Yes.)`
				`	(Not right now.)`
					defer
				`	(No.)`
					decline

> 			`	The poster includes small writing with contact instructions for inquiries about the terraforming process and Tundra's climate. Once you get through to a person on the other side, there is initially reluctance to put you through to anyone involved with the terraforming directly, but after verifying that you worked with them to get the process started, you are connected to Amy.`

Reviewer: The second sentence has two instants of "through." Not sure exactly how to fix that, though.

Author: Do you think it actually makes it worse to read?

Reviewer: Well, it uses "you" "through" in quick succession. I'm not saying it's positively wrong, but it might be better to change. Something like this could be better.
```suggestion
			`	The poster includes small writing with contact instructions for inquiries about the terraforming process and Tundra's climate. Once you get in touch with a person on the other side, they're initially reluctant to put you through to anyone involved with the terraforming directly, but after verifying that you worked with them to get the process started, you are connected to Amy.`
```

Author: ```suggestion
			`	The poster includes small writing with contact instructions for inquiries about the terraforming process and Tundra's climate. The person you reach first is initially reluctant to put you through to anyone involved with the terraforming directly, but after verifying that you worked with them to get the process started, you are connected to Amy.`
```

Reviewer: Yep, that's better, but with that change "directly" needs to come earlier.
```suggestion
			`	The poster includes small writing with contact instructions for inquiries about the terraforming process and Tundra's climate. The person you reach first is initially reluctant to put you through to anyone directly involved with the terraforming, but after verifying that you worked with them to get the process started, you are connected to Amy.`
```

			`	Once she arrives, you tell her about the Wanderers: that the work of their civilization is to repair worlds left in ruin by their previous inhabitants. You explain their view that those worlds have their own identities that they must collaborate with in order for terraforming to be successful.`
			`	She has a slew of questions ranging from how the Wanderers determine a planet's identity to how often asteroids are used in their work. Unfortunately, you are not able to offer much information about the specific details of how the Wanderers go about their terraforming work. Disappointed, but not defeated, Amy asks with renewed eagerness, "Could you take me to visit these aliens?"`
			`	Now that the Kor Mereti drones are peaceful, and the Kor Sestor drones have been eliminated, there is no longer an immediate threat to the Wanderers, and they will be focused much more on their work to repair the former Korath worlds. They would probably be happy to discuss some of that work with someone as enthused by the prospect as Amy.`
			choice
				`	"Alright. I'll prepare a bunk for you."`
					goto end
				`	"Sorry, that's too far out of my way."`
			action

> 				set "terraforming field trip: deferred later"

Reviewer: ```suggestion
				set "terraforming field trip: deferred for later"
```

			`	Amy's disappointment returns, but before leaving, she gives you her contact information, saying, "Well, I'd still love to come along the next time you are travelling that way."`
				defer
			label end
			`	She is delighted and explains that she will meet you at your ship later today, before leaving to attend to her work in preparation for her absence.`
				accept
			label repeat
			`You've returned to Tundra, where Amy, the terraforming researcher who is interested in visiting the Wanderers, works. Would you like to contact her to offer her the trip now?`
			choice
				`	(Yes.)`
				`	(Not today.)`
					defer
				`	(No.)`
					decline

> 			`	You use the contact information Amy provided the first time you told her about the Wanderers and inform her that you are prepared to take her on the journey now.`

Author: You're right, but I think we can make it even simpler without losing any important meaning:
```suggestion
			`	You contact Amy with the information she gave you on your previous meeting, telling her you're ready for the journey now.`
```

			`	Her fascination with the state of this world is temporarily forgotten when she exits your ship and sees the Wanderers for the first time. "I don't know what I expected them to be like, but it wasn't birds," she mumbles, staring at them from head to toe. However, she quickly returns to the reason for coming here. "Do you know how we can find someone who can answer my questions?"`
			`	Approaching one of the nearby Wanderers, you ask if they could direct you to Tema'a Iriket, the terraforming director. While they don't appear to understand the Hai language from your translator, they seem to recognize the name and gesture for you to wait in the spaceport, before flying off into the distance, presumably to fetch Iriket for you.`

mission "Terraforming Field Trip 2"
	name "Field Trip"
	description "Take Tema'a Iriket and Amy to <destination> to show her a world where the Wanderers have finished their work."
	source "Desi Seledrak"
	destination "Var' Kayi"
	passengers 2
	blocked "You need <capacity> in order to take on this mission."
	to offer
		has "Terraforming Field Trip 1: done"
	on offer
		conversation

> 			`It is not long before Iriket arrives. You introduce Amy to her, and explain why you have brought her here. "I would be [glad, honored] to answer any questions about our work," she agrees.`

Author: "Honor" is also used alongside "opportunity," "pride," and "reward", though all in different contexts.
The place you reference is discussing the Molt, which I expect to be described much more strongly than this.
On consideration, I think "honor" is perhaps too strong a word, but "glad" should help to moderate that somewhat.

> 			`	What follows is an extensive discussion about how the Wanderers go about planning the terraforming process for a world. Topics range from the earliest considerations as to what the goal for each planet should be to the role of their artificial minds in helping to monitor and guide the process in the long term. Your translator has some difficulty with the more technical aspects of the discussion, which leads Iriket to rephrase some things many times. Throughout it all, Amy is taking copious notes. Iriket also explains some of what they have been doing to improve the climate on this world.`
> 			`	"The gases [released, produced] by the factories will help the atmosphere retain more heat, which will in turn begin to melt the glaciers and return water to the [ecosystem, environment]. Over time, this will [mix, interact] with the abundant ash, helping to [restore, rejuvenate] the soils all over this world. In time, it will become [fruitful, bountiful] again."`

Reviewer: ```suggestion
			`	What follows is an extensive discussion about how the Wanderers go about planning the terraforming process for a world. Topics range from the earliest considerations as to what the goal for each planet should be to the role of their artificial minds in helping to monitor and guide the process in the long term. Your translator has some difficulty with the more technical aspects of the discussion, which leads Iriket to rephrase some things many times. Throughout it all, Amy is taking copious notes.`
			`	Iriket also explains some of what they have been doing to improve the climate on this world. "The gases [released, produced] by the factories will help the atmosphere retain more heat, which will in turn begin to melt the glaciers and return water to the [ecosystem, environment]. Over time, this will [mix, interact] with the abundant ash, helping to [restore, rejuvenate] the soils all over this world. In time, it will become [fruitful, bountiful] again."`
```
I think it flows better this way.

			`	Amy also explains about her recent experimental terraforming. "We believe the planet used to be temperate, or even tropical millions of years ago. We used an asteroid impact to trigger a supervolcano that was close to eruption. The greenhouse gases from the eruption have already begun to warm the climate."`
			`	Iriket thinks for a moment before responding. "Using the planet's [features, assets] to restore more habitable conditions is an [admirable?] approach." The approval buoys Amy, but she is somewhat deflated by what comes next. "However, an asteroid would not be our [preferred, first] approach."`
			`	As you conclude your sightseeing, Tema'a Iriket says your next stop should be <planet>, where she will provide Amy with more detailed records of their terraforming activities than she can provide orally.`

> 			`	Once Iriket has returned to her bunk, Amy turns to you. "I wonder what my professor would say about 'anthropomorphizing planets' if he saw this," she gestures towards the images of <origin> on the monitors, "and you told him they believe planets have their own identities, too."`

Reviewer: I don't think it's likely that the player would be telling the professor of a random acquaintance something in a field they don't know about.
```suggestion
			`	Once Iriket has returned to her bunk, Amy turns to you. "I wonder what my professor would say about 'anthropomorphizing planets' if he saw this," she gestures towards the images of <origin> on the monitors, "and I told him they believe planets have their own identities."`
```

Author: Rereading this, I don't think it makes sense at all.
```suggestion
			`	Once Iriket has returned to her bunk, Amy turns to you. "I wonder if any of this would change my professor's mind about the merits of 'anthropomorphizing planets'."`
```

				accept
	on complete
		dialog "When you arrive on <planet>, Tema'a Iriket tells you to meet her in the spaceport in a few hours before heading off to prepare the data for Amy."

mission "Terraforming Field Trip 4"
	name "Field Trip"

> 	description "Return Amy to <destination> following her trip to meet the Wanderers and learn about how they go about terraforming."

Reviewer: ```suggestion
	description "Return Amy to <destination> after her trip to meet the Wanderers and learn about how they go about terraforming."
```

	source "Vara K'chrai"
	destination "Tundra"
	passengers 1
	to offer
		has "Terraforming Field Trip 3: done"
	on offer
		conversation
			`Tema'a Iriket meets you in the spaceport and hands Amy a tablet, explaining, "I have stored [records, histories] of over a hundred worlds from our archive here. I hope the translation is [adequate, acceptable]. There has not been much opportunity to test our translation systems for the human language."`
			`	Amy has already begun browsing the data; Iriket's concerns about the quality of the translation seem to go unheard. "This is incredible. Thank you so much."`
			`	"Explaining our work to an outsider has been [gratifying, novel]. I must thank you for the opportunity, Captain <last>." Iriket bids you goodbye.`
			`	Amy returns to her bunk, beginning to organize the notes she has made on this trip. It seems it's time to return her to <planet>.`
				accept
	on complete

> 		log `Took Amy on a "field trip" to Wanderer space to learn about their approach to terraforming. Tema'a Iriket provided great insight and showed Amy both their ongoing and complete work, as well as providing an extensive library of Wanderer records that should keep Amy busy for a very long time.`

Reviewer: ```suggestion
		log `Took Amy on a "field trip" to Wanderer space to learn about their approach to terraforming. Tema'a Iriket provided great insight and showed Amy both the Wanderers' ongoing and complete work, as well as providing an extensive library of records that should keep Amy busy for a very long time.`
```

		conversation
			`Most of Amy's time on the return journey was spent with the Wanderer data. Even still, she claims to have barely scratched the surface of what the Wanderers have done.`
			choice
				`	"I'll leave you to your work here."`
					goto done
				`	"What did you think of the Wanderers?"`

> 			`	"Terraforming for the sake of improving ecosystems, providing a place for future inhabitants, whoever they may be, is very different to most of what I saw in the Paradise Worlds," she explains. "Many of the people there are only interested in terraforming for the economic opportunities for themselves, and they have little regard for anything that doesn't directly increase the planet's value for themselves or their clients. Even the garden or 'natural' areas are artificial; forcing the whims and desires of the rich onto nature.`

Reviewer: ```suggestion
			`	"Terraforming for the sake of improving ecosystems, providing a place for future inhabitants, whoever they may be, is very different to most of what I saw in the Paradise Worlds," she replies. "Many of the people there are only interested in terraforming for the economic opportunities for themselves, and they have little regard for anything that doesn't directly increase the planet's value for themselves or their clients. Even the gardens or 'natural' areas are artificial; they force the whims and desires of the rich onto nature.`
```

			`	"I've heard about a project in the Free Worlds working on affordable terraforming as well. Perhaps we should look into working with them. The Republic might not be too happy about that, though."`
			choice
				`	"I'm glad you had a good time."`
					goto done
				# TODO: when alternatives to the Free Worlds are available, this will need to be conditional.
				`	"I actually worked with the founder of the Conservatory. I'm sure I could introduce you."`
			`	She thanks you for the offer but declines. "You've done more than enough for now. Besides, I'm going to be busy studying for the foreseeable future."`
			label done

> 			`	Amy thanks you for the "field trip" before departing your ship.`

Reviewer: This seems a little abrupt. Could you expand it somewhat?

			`	She has a slew of questions ranging from how the Wanderers determine a planet's identity to how often asteroids are used in their work. Unfortunately, you are not able to offer much information about the specific details of how the Wanderers go about their terraforming work. Disappointed, but not defeated, Amy asks with renewed eagerness, "Could you take me to visit these aliens?"`
			`	Now that the Kor Mereti drones are peaceful, and the Kor Sestor drones have been eliminated, there is no longer an immediate threat to the Wanderers, and they will be focused much more on their work to repair the former Korath worlds. They would probably be happy to discuss some of that work with someone as enthused by the prospect as Amy.`
			choice
				`	"Alright. I'll prepare a bunk for you."`
					goto end
				`	"Sorry, that's too far out of my way."`
			action
				set "terraforming field trip: deferred later"

> 			`	Amy's disappointment returns, but before leaving, she gives you her contact information, saying, "Well, I'd still love to come along the next time you are travelling that way."`

Reviewer 3: American spelling.
```suggestion
			`	Amy's disappointment returns, but before leaving, she gives you her contact information, saying, "Well, I'd still love to come along the next time you are traveling that way."`
```
